### PR TITLE
chore(ui): fix ui consistency

### PR DESCRIPTION
**Pull request by rudolfs**
- use `display name` instead of `handle` in validations, closes: https://github.com/radicle-dev/radicle-upstream/issues/919
- hide "Session management" section in settings behind `isDev` (based on discussion in Figma)
- update onboarding success screen copy
- fix wording in project creation tooltip ("the the" -> "the")
- respect singular/plural noun forms in project stats

### CODE REVIEW

File: ui/Screen/Onboarding/EnterName.svelte [hunk 1]
```diff
@@ -73,7 +73,7 @@
     </p>
     <Input.Text
       autofocus={true}
-      placeholder="Enter a name"
+      placeholder="Enter a display name (i.e. coolprogrammer3000)"
```
Comment (NunoAlexandre) [inline ui/Screen/Onboarding/EnterName.svelte:76]:
```suggestion
      placeholder="Enter a display name (e.g. coolprogrammer3000)"
```

I think that `i.e.` stands for "that is, ..."

File: ui/Screen/Onboarding/Success.svelte [hunk 2]
```diff
@@ -48,8 +48,8 @@
     <p
       style="text-align: center; width: 23.13rem; margin: 1.75rem 0 1.75rem 0;
       color: var(--color-foreground-level-6);">
-      This is your <span class="typo-text-bold">Radicle ID</span>. Click to copy
-      it and share it with others so that they can find you.
+      This is your Radicle ID — it's unique to you!<br /> Click to copy it and share
```
Comment (NunoAlexandre) [inline ui/Screen/Onboarding/Success.svelte:51]:
```suggestion
      This is your Radicle ID — it uniquely identifies you in the Radicle Network!<br /> Click to copy it and share
```

"it's unique do you" sounds like it's unique within my ownership, not that it is unique in the network. Feel free to reformulate in any other way :)

**Review by NunoAlexandre — COMMENTED** [review]
Just a comment!

File: ui/Screen/Onboarding/Success.svelte (hunk 2 above)
Comment (rudolfs) [reply ui/Screen/Onboarding/Success.svelte:51]: I took this directly from the Figma mockup for beta 1. There was already a discussion on the exact wording, I merely copied it here. If you want to suggest changes to the wording, it probably makes sense to do it [there](https://www.figma.com/file/owmgsbs6lnUt8R1bixstCA?node-id=4147:7852#39220123). \cc @abbey-titcomb

File: ui/Screen/Onboarding/EnterName.svelte (hunk 1 above)
Comment (rudolfs) [reply ui/Screen/Onboarding/EnterName.svelte:76]: Good catch!

**Review by juliendonck — APPROVED** [review]
⚽️ -> 🥅

**Review by NunoAlexandre — APPROVED** [review]

**Pull request merged**